os/fs/driver/mtd/smart.c

```diff
@@ -304,9 +304,9 @@ struct smart_struct_s {
 #endif
 #ifdef CONFIG_MTD_SMART_JOURNALING
 	size_t journal_seq;				/* Current Sequence of Journal */
-	uint16_t njournalPerBlk;		/* Total Number of Journal data per Erase block */
-	uint16_t njournaleraseblocks;	/* Total Number of Journal Erase block */
-	uint32_t njournaldata;			/* Total Number of Journal Data */
+	uint16_t njournalPerBlk;		/* Total Number of Journal entries per Erase block */
+	uint16_t njournaleraseblocks;		/* Total Number of Journal Erase block */
+	uint32_t njournalentries;			/* Total Number of Journal Entries */
```

Comment:
could you maintain the same tab space for comments

Reply:
Ok